entrypoint.sh

```diff
@@ -12,47 +12,102 @@ useradd -g ${DOCKER_GID} --home-dir /home/node -s /bin/bash -u ${DOCKER_UID} nod
 
 chown -R node:node /home/node/.config
 chown node:node /app
+chown -R node:node /app/cdk.out
 
-echo
-echo ╭━━━┳╮╱╱╱╱╱╭╮╭╮
-echo ┃╭━╮┃┃╱╱╱╱╭╯╰┫┃
-echo ┃┃╱┃┃┃╭━━┳┻╮╭┫┃
-echo ┃╰━╯┃┃┃┃━┫╭┫┃╰╯
-echo ┃╭━╮┃╰┫┃━┫┃┃╰┳╮
-echo ╰╯╱╰┻━┻━━┻╯╰━┻╯
-echo 
-echo Configure RDS Alerts and Slack Notification
-echo 
-echo Make sure you have completed the below:
-echo 
-echo '  1. Bootstrap your environment with CDKv2.x (https://docs.aws.amazon.com/cdk/v2/guide/bootstrapping.html)'
-echo '  2. Create Slack Channel and Incoming Webhook URL (https://api.slack.com/messaging/webhooks)'
-echo '  3. Save the Webhook URL in SSM Parameter store as a SecureString (https://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-paramstore-su-create.html)'
-echo 
-read -p 'Are you ready (y/n)? ' ready
-if [[ ! ${ready} =~ ^[Yy]$ ]]; then
+if [[ $# -gt 0 ]]; then # Headless mode
+    INVALIDARGS=()
+    while [[ $# -gt 0 ]]; do
+        key="${1}"
+        case $key in
+            --profile)
+            profile="${2}"
+            shift; shift
+            ;;
+            --instances)
+            instances="${2}"
+            shift; shift
+            ;;
+            --securitygroup)
```

Comment:
I'd suggest changing the cli argument flags (not the variables) to be separated by hyphens `-` e.g. `--security-group` as their just a bit more readable and maybe change `--webhookparameter` to `--slack-webhook-url` to be more inline with the others.

Reply:
I love your comment! Updated some of them to be more readable.